src/shapes/Path.ts

```diff
@@ -411,25 +411,18 @@ export class Path<
    * @static
    * @memberOf Path
    * @param {SVGElement} element to parse
-   * @param {(path: Path) => void} callback Callback to invoke after the element has been parsed
    * @param {Partial<PathProps>} [options] Options object
    */
-  static fromElement(
-    element: SVGElement,
-    callback: (path: Path) => void,
-    options: Partial<PathProps>
-  ) {
+  static async fromElement(element: SVGElement, options: Partial<PathProps>) {
     const parsedAttributes = parseAttributes(element, this.ATTRIBUTE_NAMES);
```

Comment:
can we spread this so that `d` isn't passed into options?

Reply:
yes corret